rollup.config.js

```diff
@@ -0,0 +1,16 @@
+import typescript from '@rollup/plugin-typescript';
+import commonjs from '@rollup/plugin-commonjs';
+
+export default ['cjs', 'esm'].map(format => ({
+	input: 'index.ts',
+	output: {
+		format,
+		dir: format
+	},
+	plugins: [
+		commonjs(),
+		typescript({
+			outDir: format
+		})
+	]
+}));
```

Comment:
I think? Maybe one of the two `'.'` can be dropped

```suggestion
export default {
	input: 'index.ts',
	output: {
		format: 'esm',
		dir: '.'
	},
	plugins: [
		commonjs(),
		typescript({
			outDir: '.'
		})
	]
};
```

Reply:
We need both, one for `index.js` and one for `index.d.ts`